packages/projection-typeorm/src/entity/GovernanceAction.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { BigIntColumnOptions, OnDeleteCascadeRelationOptions } from './util';
+import { BlockEntity } from './Block.entity';
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
+
+@Entity()
+export class GovernanceActionEntity {
+  @PrimaryGeneratedColumn()
+  id?: number;
+
+  // This is required to handle rollbacks, once we have transactions projection
+  // the OnDeleteCascadeRelationOptions can be moved on txId and this column could be removed
```

Comment:
Maybe it's a good idea to have a backlog item for this.

Reply:
746af31c819 should solve this.